README.md

```diff
@@ -15,22 +15,26 @@ implementation, such as [up-transport-zenoh-cpp][zenoh-transport-repo].
 
 ### Requirements:
 - Compiler: GCC/G++ 11 or Clang 13
-- Conan : 1.59 or latest 2.X
+- pip : To install others
+- Conan(pip) : 1.59 or > 2.4.2
+- Cmake(apt) : > 3.22
+- Ninja(pip) : Latest
+- Gcovr(pip) : For UT code coverage if desired
 
 #### Conan packages
 
 Using the recipes found in [up-conan-recipes][conan-recipe-repo], build these
 Conan packages:
 
-1. [up-core-api][spec-repo] - `conan create --version 1.5.8 --build=missing up-core-api/developer`
+1. [up-core-api][spec-repo]: `conan create --version 1.5.8 --build=missing up-core-api/developer`
```

Comment:
Relevance?

Reply:
The hyphen makes it look like it's part of a command.  The colon to me shows this is the command to create the Conan build.